lib/traco/class_methods.rb

```diff
@@ -5,9 +5,10 @@ def locales_for_attribute(attribute)
     end
 
     def locale_columns(*attributes)
-      attributes.flat_map { |attribute|
+      attributes = attributes.presence || translatable_attributes
+      attributes.flat_map do |attribute|
         _locale_columns_for_attribute(attribute, fallback: LocaleFallbacks::ANY_FALLBACK)
-      }
+      end
```

Comment:
I believe this project picked Weirich-style blocks, so we can probably keep the block delimiters the way they were.

Reply:
Ok sure, will fix it!